### PR TITLE
Update read me contents

### LINKED ISSUE #40: Acknowledge the original source of the code in README

### PR DESCRIPTION
**Pull request by bigismols**
Fixes #40 #41

### CODE REVIEW

**Review by oolongwater — APPROVED** [review]
lgtm

File: docs/index.md [hunk 1]
```diff
@@ -3,19 +3,20 @@
...
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**HiredFiredPro is a desktop application for managing interviewees.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using HiredFired, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
Comment (SQ77) [inline docs/index.md:15]: Should HiredFired be HiredFiredPro instead?

File: docs/index.md [hunk 2]
```diff
@@ -3,19 +3,20 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# HiredFiredPro
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```
Comment (SQ77) [inline docs/index.md:9]: Change the CI status and codecov links as well?

**Review by SQ77 — COMMENTED** [review]
Some minor changes requested

File: docs/index.md (hunk 1 above)
Comment (bigismols) [reply docs/index.md:15]: Oh oops. Thanks for catching that!

File: docs/index.md (hunk 2 above)
Comment (bigismols) [reply docs/index.md:9]: Alright, I'll change the codecov link. Thanks Shiqi

**Review by SQ77 — APPROVED** [review]
LGTM!

**Review by oolongwater — APPROVED** [review]

**Pull request merged**